src/components/photopage/profile.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+const cropperWidth = '200px';
+const styles = theme => ({
+	container: {
+		display: 'flex',
+		flexDirection: 'column',
+		justifyContent: 'center',
+		alignItems: 'center',
+		padding: theme.spacing.unit * 2.75
+	},
+	// this cropper only crops landscape-oriented/square image
+	cropper: {
+		width: cropperWidth,
+		height: cropperWidth,
+		borderRadius: '50%',
+		overflow: 'hidden',
+		display: 'flex',
+		justifyContent: 'center'
+	},
+	photo: {
+		margin: '0 auto',
+		display: 'inline',
+		height: '100%',
+		width: 'auto'
+	},
+	name: {
+		fontFamily: ['Chivo', 'san-serif'],
+		fontSize: theme.typography.fontSize * 1.5,
+		fontWeight: 'normal',
+		margin: theme.spacing.unit,
+		// prevent long names from wrapping
+		whiteSpace: 'nowrap'
+	},
+	role: {
```

Comment:
There's some duplication here with `name`. The only thing that's different seems to be `fontSize`.

Reply:
Question (I think I have asked you this before): What is the best way to solve this?
Here are several solutions.

### 1. Extract common properties and put into another classname
```jsx
const styles = theme => ({
  common: { ... },
  different1: { ... },
  different2: { ... },
})
// in component
<p className={`{classes.common} {classes.different1}`}> ... </p>
<p className={`{classes.common} {classes.different2}`}> ... </p>
```

### 2. Put common properties in a separate object
```jsx
const styles = theme => {
  const common = { ... };
  return {
    different1: { ...common, overwrite: 'center' },
    different2: { ...common, overwrite: 'top' },
  };
}
```

Which is better, both look ugly to me. But gut feeling tells me 2 is more "efficient".